Packs/Druva/Playbooks/Druva-Ransomware-Response-User-resource-URL_README.md

```diff
@@ -1,42 +0,0 @@
-Automate response actions like quarantining effected resources or snapshots to stop the spread of ransomware and avoid reinfection or contamination spread.
-
```

Comment:
Please do not delete this file.

Reply:
This was never meant for publishing.
Hence, we need to delete this.